Makefile

```diff
@@ -193,3 +197,13 @@ $(CONTROLLER_GEN): $(LOCALBIN)
 envtest: $(ENVTEST) ## Download envtest-setup locally if necessary.
 $(ENVTEST): $(LOCALBIN)
 	test -s $(LOCALBIN)/setup-envtest || GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
+
+.PHONY: gitserver
+gitserver:
+		cd tests/e2e-gitserver && \
+		 docker buildx build \
+		--no-cache \
+		--build-arg REPO_COUNT=$(REPO_COUNT) \
+		--platform $(PLATFORMS) \
+		-t quay.io/numaio/$(GITSERVER_NAME):latest \
+		--push .
```

Comment:
Since IMAGE_NAMESPACE is [configurable](https://github.com/numaproj-labs/numaplane/pull/169/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L13), maybe this should be as well?